interfaces/Portal/src/app/services/language.service.ts

```diff
@@ -0,0 +1,96 @@
+import { Injectable } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { environment } from '../../environments/environment';
+
+export class LanguageOption {
+  key: string;
+  name: string;
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LanguageService {
+  private DEFAULT_LANGUAGE_KEY = 'en';
+  private DEFAULT_LANGUAGE_LABEL = 'ENGLISH';
```

Comment:
Is it in capitals on purpose?

```suggestion
  private DEFAULT_LANGUAGE_LABEL = 'English';
```